en/services/parachute.md

```diff
@@ -0,0 +1,158 @@
+| 16       | `PARACHUTE_ERROR_FLAGS_FC_CONNECTION_ERROR`  | There is an error with the connection between parachute and flight controller (FC).                              |
+| 32       | `PARACHUTE_ERROR_FLAGS_EFTS_CONNECTION_ERROR`| There is an error with the connection between parachute and Electrical Flight Termination System (EFTS).         |
+| 64       | `PARACHUTE_ERROR_FLAGS_POD_CONNECTION_ERROR` | There is an error with the parachute pod.                                                                        |
+| 128      | `PARACHUTE_ERROR_FLAGS_EFTS_DIAGNOSE`        | Parachute Electrical Flight Termination System (EFTS) diagnosis failed.                                          |
+| 256      | `PARACHUTE_ERROR_FLAGS_CHARGING_FAILED`      | Parachute module charging failed.                                                                                |
+| 512      | `PARACHUTE_ERROR_FLAGS_EXTERNAL_POWER_ERROR` | There is an error with the parachute external power source.                                                      |
+| 1024     | `PARACHUTE_ERROR_FLAGS_GS_CONNECTION_ERROR`  | There is an error with the connection between parachute and Ground Station (GS).                                 |
+| 2048     | `PARACHUTE_ERROR_FLAGS_GPS_ERROR`            | There is an error with the parachute's GPS.                                                                      |
+| 4096     | `PARACHUTE_ERROR_FLAGS_SUBSYSTEM_CONNECTION_ERROR` | There is an error with the connection between parachute and subsystem (e.g. remote controller, expansion board, etc.). |
+| 8192     | `PARACHUTE_ERROR_FLAGS_SUBSYSTEM_FW_ERROR`   | There is an error with the parachute subsystem firmware (e.g. wrong firmware version).                           |
+| 16384    | `PARACHUTE_ERROR_FLAGS_RESERVED_1`           | Reserved for future use.                                                                                         |
+| 32768    | `PARACHUTE_ERROR_FLAGS_RESERVED_2`           | Reserved for future use.                                                                                         |
+| 65536    | `PARACHUTE_ERROR_FLAGS_LOGGING_ERROR`        | There is an error with the parachute's internal logging system.                                                  |
+| 131072   | `PARACHUTE_ERROR_FLAGS_MODULE_RETIRED`       | This parachute module is retired (i.e. too many deployments).                                                    |
+| 262144   | `PARACHUTE_ERROR_FLAGS_GLOW_WIRE_ERROR`      | There is an error with the parachute glow wire.                                                                  |
+| 524288   | `PARACHUTE_ERROR_FLAGS_OFFBOARD_CONNECTION_ERROR` | There is an error with the MAVLink connection between parachute and offboard computer.                        |
```

Comment:
what's the difference between offboard and FC? Also, this is the only one that describes the connection as a "MAVLink connection"